anypoint-signin-aware.js

```diff
@@ -44,16 +53,26 @@ export const AnypointAuth = {
       AnypointAuth._redirectUri = val;
     }
   },
```

Comment:
Please use standard jsdoc comments, right now this file is a bit messy documentation-wise.